ambry-server/src/main/java/com/github/ambry/server/AmbryServer.java

```diff
@@ -308,7 +308,7 @@ public void startup() throws InstantiationException {
       requests = new AmbryServerRequests(storageManager, networkServer.getRequestResponseChannel(), clusterMap, nodeId,
           registry, metrics, findTokenHelper, notificationSystem, replicationManager, storeKeyFactory, serverConfig,
           diskManagerConfig, storeKeyConverterFactory, statsManager, clusterParticipants.get(0));
-      requestHandlerPool = new RequestHandlerPool(serverConfig.serverRequestHandlerNumOfThreads,
+      requestHandlerPool = new RequestHandlerPool(1,
```

Comment:
let's do it through configuration, instead of hard coding.

Reply:
done